### PR TITLE
FIX: Remove count of dwi scans according to task_id

### PR DESCRIPTION
**Pull request by josephmje**
Addresses #9

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -8,8 +8,8 @@ build/**/*
 build
 dist/**/*
 dist
-fmriprep.egg-info/**/*
-fmriprep.egg-info
+dmriprep.egg-info/**/*
```
Comment (oesteban) [inline .dockerignore]: Can you submit these infrastructure changes on a different PR? I'll get them merged quickly

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+sudo: true
+
+language: python
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+
+# Command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+install: pip install -U tox-travis
```
Comment (oesteban) [inline .travis.yml]: Deployment is done via CircleCI ATM, not sure this travis config is necessary.

If there's a reason to add travis, please split on a different PR.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,94 @@
+.PHONY: clean clean-test clean-pyc clean-build docs help
```
Comment (oesteban) [inline Makefile]: This would be useful, but on a different PR (I'd be fine to see this with the changes to `.dockerignore`)

File: README.rst [hunk 4]
```diff
@@ -49,4 +49,4 @@ segmentation, skullstripping etc.) providing outputs that can be
 easily submitted to a variety of tractography algorithms.
 
 [Documentation `dmriprep.org <https://dmriprep.readthedocs.io>`_]
-[Support `neurostars.org <https://neurostars.org/tags/fmriprep>`_]
+[Support `neurostars.org <https://neurostars.org/tags/dmriprep>`_]
```
Comment (oesteban) [inline README.rst]: Different PR

File: dmriprep/interfaces/reports.py [hunk 5]
```diff
@@ -109,24 +108,24 @@ def _generate_segment(self):
...
 
-        tasks = ''
-        if counts:
-            header = '\t\t<ul class="elem-desc">'
-            footer = '\t\t</ul>'
-            lines = ['\t\t\t<li>Task: {task_id} ({n_runs:d} run{s})</li>'.format(
-                     task_id=task_id, n_runs=n_runs, s='' if n_runs == 1 else 's')
-                     for task_id, n_runs in sorted(counts.items())]
-            tasks = '\n'.join([header] + lines + [footer])
+        # tasks = ''
```
Comment (oesteban) [inline dmriprep/interfaces/reports.py]: Please remove instead of commenting out.

File: dmriprep/utils/sentry.py [hunk 6]
```diff
@@ -0,0 +1,208 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""Stripped out routines for Sentry"""
```
Comment (oesteban) [inline dmriprep/utils/sentry.py]: I would prefer to force running with `--notrack` for now instead of setting up sentry. We'll eventually enable it, but it seems to me that we would be more interested in sharing this code with fmriprep in a small dependency and keep both in sync.

File: docs/Makefile [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
```
Comment (oesteban) [inline docs/Makefile]: infrastructure

File: setup.cfg [hunk 8]
```diff
@@ -7,7 +7,7 @@ maintainer_email = code@oscaresteban.es
 description = dMRIPrep is a robust and easy-to-use pipeline for preprocessing of diverse dMRI data.
 long_description = file:README.rst
 long_description_content_type = text/x-rst; charset=UTF-8
-license = 3-clause BSD
+license = Apache License, Version 2.0
```
Comment (oesteban) [inline setup.cfg]: setup.cfg - infrastructure

File: .travis.yml (hunk 2 above)
Comment (josephmje) [reply .travis.yml]: removed

File: dmriprep/interfaces/reports.py (hunk 5 above)
Comment (josephmje) [reply dmriprep/interfaces/reports.py]: done

File: dmriprep/utils/sentry.py (hunk 6 above)
Comment (josephmje) [reply dmriprep/utils/sentry.py]: aahh k didn't know about the `--notrack` flag

File: .github/config.yml [hunk 9]
```diff
@@ -25,4 +25,4 @@ newPRWelcomeComment: >
   Of course, if you want to opt-out this time there is no
   problem at all with adding your name later.
   You will be always welcome to add it in the future whenever
-  you feel it should be listed.
\ No newline at end of file
+  you feel it should be listed.
```
Comment (oesteban) [inline .github/config.yml]: Please leave out for another PR. Happy to review a housekeeping PR with these kinds of stylistic changes only.

File: setup.cfg [hunk 10]
```diff
@@ -23,15 +23,16 @@ install_requires =
     indexed_gzip >=0.8.8
     nibabel >=2.2.1
     nilearn !=0.5.0, !=0.5.1
-    nipype >=1.2.0
```
Comment (oesteban) [inline setup.cfg]: Are any updates to setup.cfg strictly necessary for this PR? I assume they aren't.

Comment (josephmje) [reply setup.cfg]: reverted them back

**Comment by oesteban** [comment]
Before merging, @garikoitz could you check whether this indeed solves #9 ?

**Comment by garikoitz** [comment]
Just checked, it is still running but it went through the previous error so in my end is good to go
